lib/Backend/Commands.hs

```diff
@@ -319,6 +319,10 @@ buildCopyOperations
 buildCopyOperations oldBackend newBackend oldQPath@(QualifiedPath oldBackendNameMb oldPath) (QualifiedPath newBackendNameMb newPath) force = do
   entryOrDir <- getEntryOrDirThrow oldBackend CPRPathNotFound oldQPath
 
+  -- Don't need to copy directory or entry to itself
+  when ((oldBackendNameMb == newBackendNameMb) && (oldPath == newPath)) $
```

Comment:
We could simplify this check like this
````haskell
when (oldQPath == newQPath) do
...
````

Reply:
Done